lib/location/method_cellular.c

```diff
@@ -228,8 +222,8 @@ int method_cellular_init(void)
 		    method_cellular_positioning_work_fn);
 	lte_lc_register_handler(method_cellular_lte_ind_handler);
 
-#if !defined(CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL)
-	int ret = multicell_location_provision_certificate(false);
+#if defined(CONFIG_LOCATION_SERVICE_HERE)
+	int ret = location_service_utils_provision_ca_certificates();
 
 	if (ret) {
 		LOG_ERR("Certificate provisioning failed, ret %d", ret);
```

Comment:
**nit** Can't add a comment to the right line, but on line 231 the status code in the warning trace is incorrect, remove the extra "S".